LatinumNetwork/mosaic
# [BRAN-155]-Create-card-header-component

## Purpose/Description:

Added CardHeader component to Mosaic

### Screenshots:
![Captura de pantalla 2023-12-12 a la(s) 4 15 11 p m](https://github.com/LatinumNetwork/mosaic/assets/45240616/42d193e5-6b3f-4b39-8075-ec0fbb8f1ccd)

## Jira Link:

[BRAN-155](https://collagegroup.atlassian.net/browse/BRAN-155?atlOrigin=eyJpIjoiNjQ4ZmQyZDBiYjNjNDNkZGE1MjcwNTM0N2I5ZjUwZmMiLCJwIjoiaiJ9)

### Reviewer Instructions:

- Pull this branch
- Run` npm run storybook`
- You should see the CardHeader item in the menu


[BRAN-155]: https://collagegroup.atlassian.net/browse/BRAN-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
